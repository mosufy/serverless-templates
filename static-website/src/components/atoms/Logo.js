import React from "react";

import logo from "assets/img/logo.svg";

const Logo = () => {
  return <img src={logo} className="App-logo" alt="logo" />;
};

export default Logo;
