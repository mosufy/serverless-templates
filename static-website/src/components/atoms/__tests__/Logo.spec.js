import React from "react";
import { shallow } from "enzyme";

import Logo from "Components/atoms/Logo";

describe("test Logo component", () => {
  it("should render correctly", () => {
    const component = shallow(<Logo />);
    expect(component).toMatchSnapshot();
  });
});
