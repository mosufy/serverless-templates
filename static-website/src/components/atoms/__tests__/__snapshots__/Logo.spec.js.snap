// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Logo component should render correctly 1`] = `
<img
  alt="logo"
  className="App-logo"
  src="logo.svg"
/>
`;
